dev-utils/src/commands/test_vector_command.rs

```diff
@@ -286,35 +286,40 @@ impl VectorFactory {
         let sign_key = self.wallet.transfer_addresses[0].2.clone();
         let utxo = TxoPointer::new(withdraw_unbonded_tx_id, 0);
         let staking_address = self.wallet.staking_address.clone().unwrap().0;
-        let tx = DepositBondTx::new(
-            vec![utxo],
-            staking_address,
-            StakedStateOpAttributes::new(self.chain_hex_id),
-        );
+        let attributes = StakedStateOpAttributes::new(self.chain_hex_id);
+        let tx = DepositBondTx::new(vec![utxo], staking_address, attributes);
         let proof = TestVectorWallet::gen_proof(public_key)?.unwrap();
         let witness: TxWitness = vec![TxInWitness::TreeSig(
             sign_key.schnorr_sign(&Transaction::DepositStakeTransaction(tx.clone()))?,
             proof,
         )]
         .into();
+        let tx_id = tx.id();
 
         let deposit_vector = DepositStakeVector {
             staking_address: format!("{}", staking_address),
             witness: hex::encode(witness.encode()),
             transaction: hex::encode(tx.encode()),
+            tx_id: hex::encode(tx_id),
         };
         self.test_vectors.deposit_stake_vector = Some(deposit_vector);
         Ok(())
     }
 
     fn create_nodejoin_tx(&mut self) -> Result<()> {
         let (staking_address, _, sign_key) = self.wallet.staking_address.clone().unwrap();
-        let tendermint_validator_pubkey = TendermintValidatorPubKey::Ed25519([1; 32]);
+        let pk = hex::decode("d75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511a")
+            .unwrap();
```

Comment:
how about "expect"?

Reply:
it's dev-utils and the hexadecimal payload is coming from ed25519 test vectors